web/src/pages/Tag.jsx

```diff
@@ -67,11 +73,8 @@ export function Tag() {
       navigate("/?" + params.toString()); // force jump to Status page
       return;
     }
-    if (dependencies === undefined) {
-      dispatch(getDependencies({ pteamId: pteamId, serviceId: serviceId }));
-      return;
-    }
-    if (!tagId || !currentTagDependencies?.length > 0) {
+
+    if (!tagId || (!currentTagDependencies?.length > 0 && !serviceDependenciesIsLoading)) {
```

Comment:
currentTagDependencies は必ず配列のはずなので、0配列長が0判定で問題なさそうに見えます
```suggestion
    if (!tagId || (currentTagDependencies.length == 0 && !serviceDependenciesIsLoading)) {
```